src/python-common/ceph/deployment/translate.py

```diff
@@ -46,15 +49,17 @@ def run(self):
 
             cmd += " --filestore"
 
-        # HORRIBLE HACK
-        if self.spec.objectstore == 'bluestore' and \
-           not self.spec.encrypted and \
-           not self.spec.osds_per_device and \
-           len(data_devices) == 1 and \
-           not db_devices and \
-           not wal_devices:
-            cmd = "lvm prepare --bluestore --data %s --no-systemd" % (' '.join(data_devices))
-            return cmd
+        # # HORRIBLE HACK
+        # if self.spec.objectstore == 'bluestore' and \
+        #    not self.spec.encrypted and \
+        #    not self.spec.osds_per_device and \
+        #    len(data_devices) == 1 and \
+        #    not db_devices and \
+        #    not wal_devices:
+        #     cmd = "lvm prepare --bluestore --data %s --no-systemd" % (' '.join(data_devices))
+        #     if self.osd_id_claims:
+        #         cmd += " --osd-id {}".format(" ".join(self.osd_id_claims))
+        #     return cmd
```

Comment:
This will likely break Teuthology, right?

Reply:
I'm about to find, teuthology run is about to get scheduled